src/index.html

```diff
@@ -11,6 +11,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="search search--big" data-qa="big">
+      <div class="search--bigimage"></div>
```

Comment:
Use a semantic tag for image

Reply:
In the rules of the task it was written:
Icon implemented using background-image CSS property

Do I need to do this via img?